lib/src/utils.dart

```diff
@@ -1,4 +1,130 @@
 import 'package:charcode/charcode.dart';
+import 'nodes.dart';
+
+/// Defines HTML elements which are not supported by this parser.
+const List<String> unsupportedTags = const [
+  '!DOCTYPE',
+];
+
+/// Defines `void` HTML elements (no closing tag).
+Set<String> voidTags = new Set.from([
+  'area',
+  'base',
+  'basefont',
+  'bgsound',
+  'br',
+  'col',
+  'command',
+  'embed',
+  //'frame', -- is this still a valid HTML5 tag?
+  'hr',
+  'img',
+  'input',
+  'keygen',
+  'link',
+  'meta',
+  'param',
+  'source',
+  'track',
+  'wbr',
+]);
+
+/// Defines normal HTML elements (requires closing tags).
+Set<String> regularTags = new Set.from([
```

Comment:
I am actually not sure this is a good idea, since Angular/web components can always override an existing directive so this is something Angular can enforce versus here.